### PR TITLE
WT-7484 Fixed uninitialized value and removed unused values

### PR DESCRIPTION
**Pull request by etienneptl**

### CODE REVIEW

**Comment by quchenhao** [comment]
Can the title be more specific?

**Review by quchenhao — APPROVED** [review]
The change LGTM

File: test/cppsuite/test_harness/workload/thread_context.h [hunk 1]
```diff
@@ -201,7 +201,7 @@ class thread_context {
      * _max_op_count is the current maximum number of operations that can be executed in the current
      * transaction. _max_op_count will always be <= _max_op.
      */
-    int64_t _min_op, _max_op, _max_op_count;
+    int64_t _min_op, _max_op, _max_op_count = 0;
```
Comment (quchenhao) [inline test/cppsuite/test_harness/workload/thread_context.h:204]: In wiredtiger code convention, we separate declaration and initialization.

Comment (etienneptl) [reply test/cppsuite/test_harness/workload/thread_context.h:204]: Declaration and initialisation have been done simultaneously in many places in this project as we decided it was fine this way. @luke-pearson , should we have official coding guidelines for c++ syntax ?

**Pull request merged**